### PR TITLE
Fixed caching

### PR DESCRIPTION
**Pull request by RPrudden**

### CODE REVIEW

File: decision_lambda.py [hunk 1]
```diff
@@ -10,6 +10,7 @@
 from __future__ import print_function
 
 import math
+import pickle
```
Comment (niallrobinson) [inline decision_lambda.py]: no pickle required

File: tests/lambda_tests.py [hunk 2]
```diff
@@ -7,16 +8,21 @@
 from decision_lambda import *
 
 class LambdaDecisionTest(unittest.TestCase):
-	base = os.path.split(__file__)[0]
+    base = os.path.split(__file__)[0]
```
Comment (niallrobinson) [inline tests/lambda_tests.py]: `os.getcwd()`

File: tests/lambda_tests.py [hunk 3]
```diff
@@ -7,16 +8,21 @@
...
 
-	evtfile = open(os.path.join(base, 'sample_event.json'), 'r')
-	event = json.loads(evtfile.read())
-	evtfile.close()
+    evtfile = open(os.path.join(base, 'sample_event.json'), 'r')
```
Comment (niallrobinson) [inline tests/lambda_tests.py]: with open

File: tests/lambda_tests.py (hunk 2 above)
Comment (RPrudden) [inline tests/lambda_tests.py]: os.getcwd() doesn't get the file dir

**Pull request merged**